app/views/pets/show.html.erb

```diff
@@ -5,26 +5,28 @@
     <li><%= link_to 'Change to Adoptable', "/pets/#{@pet.id}/adoptable", method: :patch %></li>
   <% end %>
 </ul>
-<h1><%= link_to @pet.name, "/pets/#{@pet.id}" %></h1>
-
-<% if !favorite_pets.favorited?(@pet.id) %>
-  <%= button_to 'Add to Favorites', "/favorites/#{@pet.id}", method: :patch %>
-<% else %>
-  <%= button_to 'Remove from Favorites', "/favorites/#{@pet.id}", method: :delete %>
-<% end %>
-<% if @pet.application_approved? %>
-  <h2>On hold for <%= @pet.find_approved_application.name %></h2>
-<% end %>
-<ul>
-  <li><%= @pet.age %></li>
-  <li><%= @pet.sex %></li>
-  <li><%= @pet.description %></li>
-  <li><%= @pet.status %></li>
-</ul>
-<%= link_to 'View All Applications for This Pet', "/pets/#{@pet.id}/applications" %>
+<h1 class="title"><%= link_to @pet.name, "/pets/#{@pet.id}" %></h1>
+<div id="pet-show"
 <p><img src=<%= @pet.image %> /></p>
-
-<ul class="menu">
-  <li><%= link_to 'Update Pet', "/pets/#{@pet.id}/edit" %></li>
-  <li><%= link_to 'Delete Pet', "/pets/#{@pet.id}", method: :delete %></li>
-</ul>
+  <% if !favorite_pets.favorited?(@pet.id) %>
+    <%= button_to 'Add to Favorites', "/favorites/#{@pet.id}", method: :patch %>
+  <% else %>
+    <%= button_to 'Remove from Favorites', "/favorites/#{@pet.id}", method: :delete %>
+  <% end %>
+  <% if @pet.application_approved? %>
+    <h2>On hold for <%= @pet.find_approved_application.name %></h2>
+  <% end %>
+  <ul class="pet-show">
```

Comment:
here i mostly centered everything and moved the image to be underneath the pet's name